### DISCUSSION TITLE
Amps view

### DISCUSSION THREAD

**Discussion by maindrive** [emporia-vue-local/esphome#231] [Ideas]
How hard would it be to add Amps for each CT and total Amps?

**Comment by flaviut** [comment]
There's a bit more discussion about this at https://github.com/emporia-vue-local/esphome/discussions/230

**Reply by maindrive** [reply]
can't find where phase_a_voltage has a state, I see phase_a but not an integer

**Reply by maindrive** [reply]
- platform: template
    name: "Phase A Amps"
    lambda: return id(phase_a_power).state / id(phase_a).state;
    update_interval: 1s
    id: total_phase_a_current
    device_class: current
    state_class: measurement
    unit_of_measurement: "A"

  - platform: template
    name: "Phase B Amps"
    lambda: return id(phase_b_power).state / id(phase_b).state;
    update_interval: 1s
    id: total_phase_b_current
    device_class: current
    state_class: measurement
    unit_of_measurement: "A"

**Reply by maindrive** [reply]
/config/esphome/emporia2-energy-monitor.yaml:257:46: error: 'class esphome::emporia_vue::PhaseConfig' has no member named 'state'
     lambda: return id(phase_a_power).state / id(phase_a).state;
                                              ^~~~~
/config/esphome/emporia2-energy-monitor.yaml: In lambda function:
/config/esphome/emporia2-energy-monitor.yaml:266:46: error: 'class esphome::emporia_vue::PhaseConfig' has no member named 'state'
     lambda: return id(phase_b_power).state / id(phase_b).state;
                                              ^~~~~

**Reply by flaviut** [reply]
I didn't close this to avoid helping, but just to try and keep the issue in one place so it's easier for me to manage :)

Can you please move your question there?

**Reply by juavinas** [reply]
> * name: "Phase A Amps

Where to write these lines, getting errors.
